diff --git a/CMakeLists.txt b/CMakeLists.txt
index fcf034b1..fec8546c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,6 +6,18 @@ get_package_version(PACKAGE_VERSION PROJECT_VERSION)
 
 project(Opus LANGUAGES C VERSION ${PROJECT_VERSION})
 
+if(OPUS_DNN_DATA_DIR)
+    if(EXISTS "${OPUS_DNN_DATA_DIR}/dnn")
+        file(COPY "${OPUS_DNN_DATA_DIR}/dnn/"
+            DESTINATION "${CMAKE_CURRENT_SOURCE_DIR}/dnn")
+    elseif(EXISTS "${OPUS_DNN_DATA_DIR}/fargan_data.h")
+        file(COPY "${OPUS_DNN_DATA_DIR}/"
+            DESTINATION "${CMAKE_CURRENT_SOURCE_DIR}/dnn")
+    else()
+        message(FATAL_ERROR "OPUS_DNN_DATA_DIR='${OPUS_DNN_DATA_DIR}' has no dnn/ data")
+    endif()
+endif()
+
 include(OpusFunctions)
 include(OpusBuildtype)
 include(OpusConfig)
