From 0f07fbfc46e014e61cf31ab7e65fb087b5055d4c Mon Sep 17 00:00:00 2001
From: "David C. Manuelda" <StormByte@gmail.com>
Date: Fri, 2 Jan 2026 16:55:23 +0100
Subject: [PATCH] libvmaf: Test for the correct symbol (following upstream)

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 25aa2f84b2f..2cf2370d907 100644
--- a/meson.build
+++ b/meson.build
@@ -968,7 +968,7 @@ check_pkg_config = [
   ['libtls', ['tls.h'], ['tls_configure'], {}],
   ['libv4l2', ['libv4l2.h'], ['v4l2_ioctl'], {}],
   ['libvidstab', ['vid.stab/libvidstab.h'], ['vsMotionDetectInit'], {'pkg': 'vidstab', 'version': '>= 0.98'}],
-  ['libvmaf', ['libvmaf.h'], ['compute_vmaf'], {'version': '>= 2.0.0'}],
+  ['libvmaf', ['libvmaf.h'], ['vmaf_init'], {'version': '>= 2.0.0'}],
   ['libvorbis', ['vorbis/codec.h'], ['vorbis_info_init'], {'pkg': 'vorbis'}],
   ['libvorbisenc', ['vorbis/vorbisenc.h'], ['vorbis_encode_init'], {'pkg': 'vorbisenc'}],
   ['libvpl', ['mfxvideo.h', 'mfxdispatcher.h'], ['MFXLoad'], {'pkg': 'vpl', 'version': '>= 2.6', 'c_args': ['-DMFX_DEPRECATED_OFF']}],
-- 
GitLab

