From 6f2144bea19a71a1479bcb0a8f7ead3808ee8099 Mon Sep 17 00:00:00 2001
From: "David C. Manuelda" <StormByte@gmail.com>
Date: Wed, 24 Dec 2025 15:10:49 +0100
Subject: [PATCH 2/2] Added include needed for gcc 15

---
 source/dynamicHDR10/json11/json11.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/source/dynamicHDR10/json11/json11.cpp b/source/dynamicHDR10/json11/json11.cpp
index 762577735..236faac5e 100644
--- a/source/dynamicHDR10/json11/json11.cpp
+++ b/source/dynamicHDR10/json11/json11.cpp
@@ -23,6 +23,7 @@
 #include <cassert>
 #include <cmath>
 #include <cstdlib>
+#include <cstdint>
 #include <cstdio>
 #include <limits>
 
-- 
2.52.0

