From b8592788f5a48eccd02010ec123e232a2d4e821f Mon Sep 17 00:00:00 2001
From: "David C. Manuelda" <StormByte@gmail.com>
Date: Thu, 1 Jan 2026 12:47:43 +0100
Subject: [PATCH] Only add arm asm files when target is arm

---
 vpx_dsp/meson.build | 23 ++++++++++++++---------
 1 file changed, 14 insertions(+), 9 deletions(-)

diff --git a/vpx_dsp/meson.build b/vpx_dsp/meson.build
index 07f5b5932..b2519380b 100644
--- a/vpx_dsp/meson.build
+++ b/vpx_dsp/meson.build
@@ -660,21 +660,26 @@ if features.get('vp9', false) # MANUAL
 				# loop filters
 				'x86/highbd_loopfilter_sse2.c'
 			),
-			'neon' : dsp_optional_sources['neon'] + files(
-				'arm/highbd_vpx_convolve_copy_neon.c',
-				'arm/highbd_vpx_convolve_avg_neon.c',
-				'arm/highbd_vpx_convolve8_neon.c',
-				'arm/highbd_vpx_convolve_neon.c',
-				# loop filters
-				'arm/highbd_loopfilter_neon.c',
-			),
 			'sve': dsp_optional_sources['sve'] + files(
 				'arm/highbd_vpx_convolve8_sve.c',
 			),
 			'sve2': dsp_optional_sources['sve2'] + files(
 				'arm/highbd_vpx_convolve8_sve2.c',
-			)
+			),
 		}
+		# Only add ARM NEON sources when building for ARM targets
+		if features.get('arm', false) or features.get('aarch64', false)
+			dsp_optional_sources += {
+				'neon' : dsp_optional_sources['neon'] + files(
+					'arm/highbd_vpx_convolve_copy_neon.c',
+					'arm/highbd_vpx_convolve_avg_neon.c',
+					'arm/highbd_vpx_convolve8_neon.c',
+					'arm/highbd_vpx_convolve_neon.c',
+					# loop filters
+					'arm/highbd_loopfilter_neon.c',
+				),
+			}
+		endif
 	endif
 
 	if features.get('neon_asm', false)
-- 
2.52.0

